static/downloadables/grep_file.py

```diff
@@ -45,47 +45,71 @@ def createFilteredFile(inputFilePath, fileHash, requiredParamsHash, superPop, re
 
 
 def getFilesAndPaths(fileHash, requiredParamsHash, superPop, refGen, isRSids, timestamp, useGWASupload):
+    isFiltered = False
```

Comment:
What is the difference between isFiltered and isFilters?

Reply:
Thanks, that's been changed to isFilters.